cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -391,9 +402,7 @@ pub mod pallet {
 				UpwardMessages::<T>::put(&up[..num as usize]);
 				*up = up.split_off(num as usize);
 
-				// Send the core selector UMP signal. This is experimental until relay chain
-				// validators are upgraded to handle ump signals.
-				#[cfg(feature = "experimental-ump-signals")]
```

Comment:
Didn't we also used this to guard until enabling RFC103 on the relay chain?

Reply:
it's important that they know at least to ignore the signals if the feature is disabled (which I'm pretty sure was included in a node release already, will check)